lib/ret_web/channels/hub_channel.ex

```diff
@@ -357,6 +361,21 @@ defmodule RetWeb.HubChannel do
 
   def handle_out("mute", _payload, socket), do: {:noreply, socket}
 
+  defp handle_entry_mode_change(socket, entry_mode) do
+    hub = socket |> hub_for_socket
+    account = Guardian.Phoenix.Socket.current_resource(socket)
+
+    if account |> can?(close_hub(hub)) do
+      hub
+      |> Hub.changeset_for_entry_mode(entry_mode)
+      |> Repo.update!()
+      |> Repo.preload(@hub_preloads, force: true)
```

Comment:
Why was the `force` necessary?

Reply:
copy pasta, can see if i can understand why it was done elsewhere.

Reply:
Ah, I think I remember why I put the force in in the original code -- in that instance we were actually changing the scene association, so the preload update had to be forced, but I think the force isn't necessary here since you're only modifying a field on the hub itself.